File: svc-loan/src/main/java/com/lending/loan/controllers/ProspectController.java
package com.lending.loan.controllers;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

import com.google.protobuf.Any;
import com.lending.proto.Business;
import com.lending.proto.Person;
import com.lending.proto.Prospect;

@RestController
public class ProspectController {
    private static final Logger log = LoggerFactory.getLogger(ProspectController.class);

    @RequestMapping(value="/prospect/{firstName}", method=RequestMethod.GET, produces={"application/x-protobuf", "application/json"})
    public Prospect lookupProspect(@PathVariable String firstName) {
        Person person = Person.newBuilder().setFirstName("Michael").setLastName("Jordan").build();
        Prospect prospect = Prospect.newBuilder()
                .setPerson(person)
                .setBusiness(
                        Business.newBuilder().setName("Nocode Inc").setFein("123456").build())
                .setAnyProspect(Any.pack(person))
                .build();
        
        return prospect;
    }
}

File: svc-payment/app/controllers/v1/prospect_controller.rb
require 'google/protobuf/well_known_types'

module V1
  class ProspectController < ApplicationController

    def lookupProspect

      business = Com::Lending::Proto::Business.new(name: "NocodeInc", fein: "123345")
      person = Com::Lending::Proto::Person.new(firstName: "Joe", lastName: "Jackson")
      prospect = Com::Lending::Proto::Prospect.new(business: business)
      prospect.person = person
      prospectAny = Google::Protobuf::Any.new
      prospectAny.pack(person)
      prospect.anyProspect = prospectAny
      respond_to do |format|
        format.json { render json: prospect }
        format.protobuf { render plain: prospect.to_proto, content_type: 'application/x-protobuf' }
      end
    end
  end
end